browser/extensions/api/brave_extensions_api_client.h

```diff
@@ -11,6 +11,8 @@
 
 namespace extensions {
 
+bool IsBraveSecurityUrl(const GURL& url);
```

Comment:
I think this should probably be an anonymous function in the .cc file. It can be tested through `ShouldHideBrowserNetworkRequest`